database/boltdb.go

```diff
@@ -0,0 +1,135 @@
+package database
+
+import (
+	"bytes"
+	"errors"
+	bolt "github.com/coreos/bbolt"
+	"sort"
+)
+
+// BoltDataStore contains methods that deal with BoltDB
+type BoltDataStore struct {
+	Db *bolt.DB
+}
+type byteSlice [][]byte
```

Comment:
I would move this closer to the methods defined on that type :)

Reply:
Sure.